doc/src/introduction.tex

```diff
@@ -1,34 +1,35 @@
 \section{Introduction}
 \label{sec:Introduction}
 
-TEAL (Tool for Economic AnaLysis) is RAVEN plugin aimed to contain and deploy economic analysis for RAVEN workflows.
+TEAL (Tool for Economic AnaLysis) is a RAVEN plug-in that leverages Uncertainty Quantification, Probabilistic Risk Assessment, Parameter Optimization
+and Data Analysis framework in RAVEN to deploy complex economic analyses.
```

Comment:
This ... might be misleading. TEAL is great, but it doesn't really enable complex analyses; it just provides the model containing some economics "physics"; in contrast to HERON, that automatically builds complex RAVEN workflows. Perhaps this sentence could be rephrased to emphasize this relationship (providing economics tools to RAVEN for use in workflows).

Reply:
Admittedly, workflow generators like HERON weren't possible when this was originally written.

Reply:
Changed to "to perform basic cash flow analyses"